libexec/cli/shell.exec

```diff
@@ -62,6 +59,14 @@ if [ -z "${SINGULARITY_IMAGE:-}" ]; then
     exit 0
 fi
 
+# Does the user want to target an application? 
+# shell --> appshell
+if [ ! -z "${SINGULARITY_APPNAME:-}" ]; then
```

Comment:
it shells right into the app folder! If a user wants to work there, inspect, etc.